src/app/modules/gb-data-selection-module/table-components/gb-table-table/gb-table-table.component.ts

```diff
@@ -0,0 +1,26 @@
+import {Component, OnInit} from '@angular/core';
+import {TableService} from '../../../../services/table.service';
+import {Col} from '../../../../models/table-models/col';
+import {Row} from '../../../../models/table-models/row';
+
+@Component({
+  selector: 'gb-table-table',
+  templateUrl: './gb-table-table.component.html',
+  styleUrls: ['./gb-table-table.component.css']
+})
+export class GbTableTableComponent implements OnInit {
```

Comment:
I am not sure if it's on purpose, but maybe better to rename it to just GbTableComponent?

Reply:
yes I will rename it! but to differentiate it from the existing gb-table, I will name it to gb-table-view